docs/website/netcdf-java/reference/FeatureDatasets/CFpointImplement.html

```diff
@@ -131,7 +131,7 @@ <h3>Step 4: Test a sample file</h3>
     <p>Write a test file with some sample data in it.</p>
   </li>
   <li>
-    <p>Test in ToolsUI 4.3 (<a href="https://www.unidata.ucar.edu/software/netcdf-java/v4.6/webstart/netCDFtools.jnlp">webstart link</a>)</p>
+    <p>Test in ToolsUI 4.3 (<a href="https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/webstart/netCDFtools.jnlp">webstart link</a>)</p>
```

Comment:
Let's just reference the URL on the docs server directly and not rely on the redirect we will put into place.

Reply:
webstart links updated to point to downloads.unidata.ucar.edu.